saraho1123/currency-exchange
# Troubleshoot/conditional

## Is this a fix or a feature?
fix

## What does it fix?
pass arguments to the consolidateData method and it fixed everything!
** I know I need to come back and explain this better, but I am really tired and I can't think, but I know I need to start this!

## Where should the reviewer start?
line 32 and 41 of App.js

## How should this be tested?
Card data will not be one card behind anymore!
+ clone down
+ cd into project
+ run `npm install` in terminal
+ run `npm start` in terminal
Form should appear. It can be filled out and button clicked, and user taken to `/currency-cards`.
The data from the user inputs, api data and manipulated api data would render
This can be done multiple times by click 'Get A New Currency Exchange' at the top of the page
NOTE: styling and organization of data is incomplete at this point.

## Relevant Tickets
#4
